tests/e2e/linalg_ext_ops/BUILD

```diff
@@ -81,6 +82,14 @@ iree_check_single_backend_test_suite(
     target_backend = "cuda",
 )
 
+iree_cmake_extra_content(
+    content = """
+# Failing on Emscripten: https://github.com/iree-org/iree/pull/12041#discussion_r1101777536
```

Comment:
Can we make this into a real issue? GitHub even has a handy button to do this from a comment context menu